auth/db/README.md

```diff
@@ -0,0 +1,131 @@
+# INSERTS
+
+### Insert new platform to platform relation
+```sql
+-- Template
+INSERT INTO platform (name) VALUES ('<platformNameUppercase>');
+
+-- Example
+INSERT INTO platform (name) VALUES ('FRIDGE');
+```
+
+### Insert Steam user into users relation
+```sql
+-- Template
+INSERT INTO users (id, platform, username, realname, profileurl, avatar)
+VALUES ('<steamID>', 0, '<username>', '<realName>', '<profileUrl>', '<avatar>');
+
+-- Example
+INSERT INTO users (id, platform, username, realname, profileurl, avatar)
+VALUES ('gabelogannewell', 0, 'Rabscuttle', 'Gabe Newell', 'https://steamcommunity.com/id/gabelogannewell', 'https://avatars.cloudflare.steamstatic.com/c5d56249ee5d28a07db4ac9f7f60af961fab5426_full.jpg');
+```
+
+### Insert Epic user into users relation
+```sql
+-- Template
+INSERT INTO users (id, platform, username, preflang)
+VALUES ('<accountID>', 1, '<username>', '<preferredLanguage>');
+
+-- Example
+INSERT INTO users (id, platform, username, preflang)
+VALUES ('novarchite', 1, 'Novarchite', 'en-US');
```

Comment:
For the pref language, I am not sure what epic uses, but do we want to adhere to one of the [ISO 63X standards?  ](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)